obsreport/obsreport.go

```diff
@@ -84,7 +93,8 @@ func buildComponentPrefix(componentPrefix, configType string) string {
 }
 
 // AllViews return the list of all views that needs to be configured.
-func AllViews() (views []*view.View) {
+func AllViews() *ObsMetrics {
```

Comment:
I think nobody calls this except Configure, so can be made private.

Reply:
It was also used in `obsreport_test` package. I'll see how I can make it private and solve the dependency cycle at the same time.

Reply:
done